### ISSUE TITLE
After updating to version TFS artifacts for Release Management 15.223.1 DownloadExternalBuildArtifacts no longer works. 

### ISSUE THREAD

**Issue by Makar4iq** [microsoft/azure-pipelines-extensions#1157]
2023-06-08T08:26:34.8228367Z Error: Cannot find module 'vsts-task-lib/task'
2023-06-08T08:26:34.8229485Z     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
2023-06-08T08:26:34.8230286Z     at Function.Module._load (internal/modules/cjs/loader.js:562:25)
2023-06-08T08:26:34.8230795Z     at Module.require (internal/modules/cjs/loader.js:692:17)
2023-06-08T08:26:34.8231181Z     at require (internal/modules/cjs/helpers.js:25:18)
2023-06-08T08:26:34.8231956Z     at Object.<anonymous> (C:\azagent\A2\_work\_tasks\DownloadExternalBuildArtifacts_b099689b-039e-4450-8658-c72e3895dd3f\15.223.1\download.js:14:12)
2023-06-08T08:26:34.8232839Z     at Module._compile (internal/modules/cjs/loader.js:778:30)
2023-06-08T08:26:34.8233234Z     at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
2023-06-08T08:26:34.8233552Z     at Module.load (internal/modules/cjs/loader.js:653:32)
2023-06-08T08:26:34.8233854Z     at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
2023-06-08T08:26:34.8234166Z     at Function.Module._load (internal/modules/cjs/loader.js:585:3)

![image](https://github.com/microsoft/azure-pipelines-extensions/assets/42178873/1348dba1-4a37-451f-8259-27a8da9e282d)

**Comment by Skoucail** [comment]
Same issue for us.
As this is for release management i hope this can be handled with high prio!
For the moment we cant do any releases as downloading of artifiacts is a core component.

**Comment by shamil-mubarakshin** [comment]
Facing similar issues.
```
2023-06-08T07:33:40.4842259Z ==============================================================================
2023-06-08T07:33:40.4842474Z Task         : Download Artifacts - External Build
2023-06-08T07:33:40.4842596Z Description  : Download artifacts from external VSTS/TFS build
2023-06-08T07:33:40.4842745Z Version      : 15.223.1
2023-06-08T07:33:40.4842830Z Author       : ms-vscs-rm
2023-06-08T07:33:40.4842959Z Help         : 
2023-06-08T07:33:40.4843050Z ==============================================================================
2023-06-08T07:33:40.8863321Z module.js:478
2023-06-08T07:33:40.8863718Z     throw err;
2023-06-08T07:33:40.8863970Z     ^
2023-06-08T07:33:40.8864039Z 
2023-06-08T07:33:40.8864275Z Error: Cannot find module 'vsts-task-lib/task'
2023-06-08T07:33:40.8864609Z     at Function.Module._resolveFilename (module.js:476:15)
2023-06-08T07:33:40.8864910Z     at Function.Module._load (module.js:424:25)
2023-06-08T07:33:40.8865281Z     at Module.require (module.js:504:17)
2023-06-08T07:33:40.8865569Z     at require (internal/module.js:20:19)
2023-06-08T07:33:40.8865972Z     at Object.<anonymous> (C:\a\_tasks\DownloadExternalBuildArtifacts_b099689b-039e-4450-8658-c72e3895dd3f\15.223.1\download.js:14:12)
2023-06-08T07:33:40.8866347Z     at Module._compile (module.js:577:32)
2023-06-08T07:33:40.8866650Z     at Object.Module._extensions..js (module.js:586:10)
2023-06-08T07:33:40.8866965Z     at Module.load (module.js:494:32)
2023-06-08T07:33:40.8867226Z     at tryModuleLoad (module.js:453:12)
2023-06-08T07:33:40.8867515Z     at Function.Module._load (module.js:445:3)
2023-06-08T07:33:40.9740481Z ##[error]Exit code 1 returned from process: file name 'C:\agent\externals\node\bin\node.exe', arguments '"C:\a\_tasks\DownloadExternalBuildArtifacts_b099689b-039e-4450-8658-c72e3895dd3f\15.223.1\download.js"'.
```

**Comment by Skoucail** [comment]
From what i can see.
PR https://github.com/microsoft/azure-pipelines-extensions/pull/1071 changed the vsts-task-lib to azure-pipelines-task-lib
But they didn't update the code to use azure-pipelines-task-lib...

**Comment by Skoucail** [comment]
Issue seems fixed in version 15.223.2

**Comment by Makar4iq** [comment]
I confirm the problem is solved.
